index.html

```diff
@@ -0,0 +1,487 @@
+
+                </p>
+                <p>Under some circumstances, a glass of water can stay liquid as the temperature falls below 32 degrees, until it is disturbed, at which point it will rapidly freeze, releasing latent heat.
+
+                </p>
+                <p>Similarly, the universe could “supercool” and stay in a unified state too long. In that case, space itself would become imbued with a mysterious latent energy.
+
+                </p>
+                <p>Inserted into Einstein’s equations, the latent energy would act as a kind of antigravity, and the universe would blow itself up. Since it was space itself supplying the repulsive force, the more space was created, the harder it pushed apart.
+
+                </p>
+                <p>What would become our observable universe mushroomed in size at least a trillion trillionfold — from a submicroscopic speck of primordial energy to the size of a grapefruit — in less than a cosmic eye-blink.
+
+                </p>
+                <p>Almost as quickly, this pulse would subside, relaxing into ordinary particles and radiation. All of normal cosmic history was still ahead, resulting in today’s observable universe, a patch of sky and stars billions of light-years across. “It’s often said that there is no such thing as a free lunch,” Dr. Guth likes to say, “but the universe might be the ultimate free lunch.”
+
+                </p>
+                <p>Make that free lunches. Most of the hundred or so models resulting from Dr. Guth’s original vision suggest that inflation, once started, is eternal. Even as our own universe settled down to a comfortable homey expansion, the rest of the cosmos will continue blowing up, spinning off other bubbles endlessly, a concept known as the multiverse.
+
+                </p>
+                <p>So the future of the cosmos is perhaps bright and fecund, but do not bother asking about going any deeper into the past.
+
+                </p>
+                <p>We might never know what happened before inflation, at the very beginning, because inflation erases everything that came before it. All the chaos and randomness of the primordial moment are swept away, forever out of our view.
+
+                </p>
+                <p>“If you trace your cosmic roots,” said Abraham Loeb, a Harvard-Smithsonian astronomer who was not part of the team, “you wind up at inflation.”
+
+                </p>
+                <span class="credit">A version of this article appears in print on March 17, 2014, on Page A1 of the New York edition with the headline:<br>Space Ripples Reveal Big Bang’s Smoking Gun. Order Reprints | Today’s Paper | Subscribe
+                </span>
+            </div>
+            <div class="text-footer">
+                <button>READ 615 COMMENTS</button>
+                <img src="./Assets/footer_icons.jpg" alt="footer media">
+            </div>
+        </div>
+        <div class="links">
```

Comment:
You should make every link in this container a real link. Kindly put all `<li>` inside `<a>`. Do this also in all the links of the `<div class="more-links">`.